apps/api/src/modules/user/user.controller.ts

```diff
@@ -16,11 +18,12 @@ export class UserController {
   constructor(private readonly userService: UserService) {}
 
   @Get('me')
+  @UseGuards(JwtAuthGuard)
   @ApiOperation({ summary: 'fetch for user data' })
   @ApiOkResponse({ type: GetUserDto })
   @ApiUnauthorizedResponse({ description: 'User is not login' })
   @ApiBadRequestResponse({ description: 'Wrong format' })
-  getUserData() {
-    return 'user data'
+  getUserData(@Req() req) {
```

Comment:
(Optional) Should we create custom decorator for user
ex. instead of `@Req() req` then req.user 
change to `@User user`